packages/test-runner-intercept/README.md

```diff
@@ -0,0 +1,78 @@
+## Usage
+
+Setup
+
+```js
+// web-test-runner.config.mjs
+import { interceptModulePlugin } from '@web/test-runner-intercept/plugin';
+
+export default {
+  plugins: [interceptModulePlugin()],
+};
+```
+
+```js
+// src/getTimeOfDay.js
+import { getCurrentHour } from 'time-library';
+
+export function getTimeOfDay() {
+  const hour = getCurrentHour();
+  if (hour < 6 || hour > 18) {
+    return 'night';
+  }
+  return 'day';
+}
+```
+
+Simple test scenario:
+
+```js
+// test/getTimeOfDay.test.js
+import { interceptModule } from '@web/test-runner-intercept';
+
+const timeLibrary = await interceptModule('time-library');
+const { getTimeOfDay } = await import('../src/getTimeOfDay.js');
+
+describe('getTimeOfDay', () => {
+    it('returns night at 2', () => {
+        timeLibrary.getCurrentHour = () => 2;
+        const result = getTimeOfDay();
+        if (result !== 'night') {
+            throw;
+        }
+    });
+});
+```
+
+More extended test scenario with common helper libraries:
+
+```js
+// test/getTimeOfDay.test.js
+import { stub } from 'sinon';
+import { expect } from 'chai';
+import { interceptModule } from '@web/test-runner-intercept';
+
+const timeLibrary = await interceptModule('time-library');
+const { getTimeOfDay } = await import('../src/getTimeOfDay.js');
+
+describe('getTimeOfDay', () => {
+  it('returns night at 2', () => {
+    const stubGetCurrentHour = stub(timeLibrary, 'getCurrentHour').returns(2);
+    try {
+      const result = getTimeOfDay();
+      expect(result).to.equal('night');
+    } finally {
+      stubGetCurrentHour.restore();
+    }
+  });
+  it('returns day at 14', () => {
+    const stubGetCurrentHour = stub(timeLibrary, 'getCurrentHour').returns(14);
+    try {
+      const result = getTimeOfDay();
+      expect(result).to.equal('day');
+    } finally {
+      stubGetCurrentHour.restore();
+    }
+  });
+});
+```
```

Comment:
I see in some of last PRs that docs are not added to the website anymore https://modern-web.dev/

cc @thepassle @koddsson 

maybe good to add some rules about that like if we should always do it and be strict with that
and if there are exceptions, good to document where not to do that?

Reply:
Although I agree it should be in the website docs, I would do this once the package has been a bit more tested. And maybe even with a 1.0.0 release.